free-courses-pt_BR.md

```diff
@@ -59,11 +61,22 @@
 * [Introdução ao Java e Orientação a objetos](https://www.udemy.com/introducao-ao-java-e-orientacao-a-objetos/) - Helder Guimaraes Aragao (Udemy)
 
 
+### Javascript
+
+* [Fast & Furious](https://www.youtube.com/watch?v=yRr8Wo4XfYY&list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts (Youtube)
+
+
 ### Kotlin
 
 * [Desenvolvedor Kotlin Iniciante](https://www.udemy.com/desenvolvedor-kotlin-iniciante/) - Gabriel Ferrari, Adriano Sacardo (Udemy)
 
 
+### Node.js
```

Comment:
If you want Node to be a subcategory of javascript, add another # and move it to the javascript section. 

Or... leave Node as its own section and put the heading after Kotlin in the contents list.

Reply:
I put Node in your own category.